hw21-jpql/src/main/java/ru/otus/demo/DbServiceDemo.java

```diff
@@ -0,0 +1,52 @@
+package ru.otus.demo;
+
+import org.hibernate.cfg.Configuration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import ru.otus.core.repository.DataTemplateHibernate;
+import ru.otus.core.repository.HibernateUtils;
+import ru.otus.core.sessionmanager.TransactionManagerHibernate;
+import ru.otus.crm.dbmigrations.MigrationsExecutorFlyway;
+import ru.otus.crm.model.Client;
+import ru.otus.crm.service.DbServiceClientImpl;
+
+public class DbServiceDemo {
+
+    private static final Logger log = LoggerFactory.getLogger(DbServiceDemo.class);
+
+    public static final String HIBERNATE_CFG_FILE = "hibernate.cfg.xml";
+
+    public static void main(String[] args) {
+        var configuration = new Configuration().configure(HIBERNATE_CFG_FILE);
+
+        var dbUrl = configuration.getProperty("hibernate.connection.url");
+        var dbUserName = configuration.getProperty("hibernate.connection.username");
+        var dbPassword = configuration.getProperty("hibernate.connection.password");
+
+        new MigrationsExecutorFlyway(dbUrl, dbUserName, dbPassword).executeMigrations();
+
+        var sessionFactory = HibernateUtils.buildSessionFactory(configuration, Client.class);
```

Comment:
```suggestion
        var sessionFactory = HibernateUtils.buildSessionFactory(configuration, Client.class, Address.class, Phone.class);
```

Для `SessionFactory` нужно передать все хранимые сущности.

Reply:
Исправил, гляньте пожалуйста еще раз